desktop/ui/src/main/java/org/datacleaner/user/DataCleanerHome.java

```diff
@@ -97,9 +103,11 @@ private static FileObject findDataCleanerHome() throws FileSystemException {
             logger.info("Running in WebStart mode. Attempting to build DATACLEANER_HOME in user.home: {} -> {}", path,
                     candidate);
         } else {
-            // in normal mode, the default folder will be in the working
-            // directory
-            candidate = manager.resolveFile(".");
+            if (candidate == null) {
```

Comment:
this if-check should not be used. We have the isUsable(...) for making sure that the candidate is good.